Wire-iOS/Sources/UserInterface/Components/Views/InviteButton.swift

```diff
@@ -25,9 +25,15 @@ final class InviteButton: IconButton {
         super.init()
 
         clipsToBounds = true
-        titleLabel?.font = FontSpec(.normal, .semibold).font
+        titleLabel?.font = FontSpec.normalSemiboldFont.font!
```

Comment:
Here are we sure that we will always get the font instead of an optional nil?
And can we write it as below ?
```suggestion
        titleLabel?.font = .normalSemiboldFont
```

Reply:
Nope,  we can't because we're trying to get rid of the that file where that property exists. That's why we're using FontSpec.normalSemiboldFont.font!

Reply:
Thanks I got it.